pkgs/by-name/li/lima/package.nix

```diff
@@ -40,6 +41,13 @@ buildGoModule (finalAttrs: {
     substituteInPlace Makefile \
       --replace-fail 'codesign -f -v --entitlements vz.entitlements -s -' 'codesign -f --entitlements vz.entitlements -s -' \
       --replace-fail 'rm -rf _output vendor' 'rm -rf _output'
+  ''
+  + lib.optionalString (!stdenv.hostPlatform.isDarwin) ''
+    substituteInPlace pkg/networks/commands.go \
+      --replace-fail '/usr/bin/pkill' '${lib.getExe' procps "pkill"}'
```

Comment:
As far as I see, Linux users do not need this change right now because that code part is skipped by `runtime.GOOS != "darwin"`.
  https://github.com/lima-vm/lima/blob/483a694d0d15ce701d89a0203360052b9d9c81af/pkg/networks/reconcile/reconcile.go#L237-L239